lib/basedriver/helpers.js

```diff
@@ -27,19 +27,14 @@ async function configureApp (app, appExt, mountRoot="Volumes", windowsShareUserN
 
   if ((app || '').substring(0, 4).toLowerCase() === 'http') {
     logger.info(`Using downloadable app '${app}'`);
-    let downloadedApp = await downloadApp(app, shouldUnzipApp ? '.zip' : appExt);
+    let downloadedApp = await downloadApp(app, appExt);
     newApp = downloadedApp.targetPath;
```

Comment:
Ah, yes. The directionality of that is backwards for my mind so I always ignore it. Alas.